packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -343,17 +358,22 @@ const createLanguageClient = async (
       );
     }
   } catch (e) {
-    languageClientUtils.setStatus(ClientStatus.Error, e);
-    let eMsg =
-      typeof e === 'string' ? e : e.message ?? nls.localize('unknown_error');
+    const err = e as unknown as Error;
```

Comment:
Use of as unknown as ... is discouraged. I would rather see type guard

```
isDuration(
  predicateOrWait: PredicateWithTimeout | Duration
): predicateOrWait is Duration {
  return (predicateOrWait as Duration).milliseconds !== undefined;
}
```

Reply:
Updated! Thanks for the feedback